src/QATools/Example/Pages/LoginPage.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * This file is part of the QA-Tools library.
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @copyright Michael Iwersen <mi.iwersen@gmail.com>
+ * @link      https://github.com/qa-tools/behat-example
+ */
+
+namespace QATools\Example\Pages;
+
+
+use QATools\QATools\HtmlElements\Element\Form;
+use QATools\QATools\HtmlElements\TypifiedPage;
+
+/**
+ * Class LoginPage.
+ *
+ * @page-url('/')
+ */
+class LoginPage extends TypifiedPage
+{
+
+	/**
+	 * The login form.
+	 *
+	 * @var Form
+	 * @find-by('id' => 'login-form')
+	 */
+	protected $loginForm;
+
+	/**
+	 * Fill the login form with given data.
+	 *
+	 * @param string $username The username.
+	 * @param string $password The password.
+	 *
+	 * @return static
+	 */
+	public function fillForm($username, $password)
```

Comment:
I see no much use in `fillForm` method alone, because you need to submit form afterwards anyway. Instead I propose to:

1. rename method to `login`
2. call `$this->loginForm->submit();` after form is filled

This way:
* to have a successful login you call (from feature context) `$this->page->login('correct', 'correct');`
* to have unsuccessful login you call (from feature context) `$this->page->login('', '');`

This is way more explicit (especially for unsuccessful login), then just submitting form without knowing what values there are in the input fields after last feature context step has executed.